### PR TITLE
Sync values from the theme in the token definition

### PR DESCRIPTION
**Pull request by victorg1991**
Some default values in the token definition of the classic theme were not the same as the one used in the CSS

Blockquote small color is defined [here](https://github.com/liferay/liferay-portal/blob/cad004434e6b19e9cea38776abe415c104f921f5/modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss#L51)

Body bg is defined [here](https://github.com/liferay/liferay-portal/blob/8fb1396b753a2583a0506423a65ea015e5c11afe/modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss#L1)

In the case of body bg color maybe we want another color? I've used what the theme is defining right now, but maybe there is a mistake there also, can you review it?

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss [hunk 1]
```diff
@@ -53,8 +53,8 @@ $theme-colors-cp: map-deep-merge(
 
 	// Body
 
-	--body-bg: $body-bg;
-	--body-color: $body-color;
+	--body-bg: #{$body-bg};
+	--body-color: #{$body-color};
```
Comment (victorg1991) [inline modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss:57]: I'm not sure why it didn't work the other way, there are a lot of usages of the other syntax like [this one](https://github.com/liferay/liferay-portal/blob/c80d8b87fa86fdfb829861a46e893d64a8825bf2/modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page-design-options/components/PageDesignOptions.scss#L8)

I'm missing something here? 🤔

Comment (victorg1991) [reply modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss:57]: @marcoscv-work

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss (hunk 1 above)
Comment (eduardoallegrini) [reply modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss:57]: I don't know exactly why but custom properties work that way 🤷

Comment (pat270) [reply modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss:57]: https://sass-lang.com/documentation/breaking-changes/css-vars should explain why we need to interpolate Sass variables in custom properties

**Comment by jbalsas** [comment]
Hey @pat270, assuming this is good to push forward, right?

**Comment by jbalsas** [comment]
ci:forward

**Comment by pat270** [comment]
@jbalsas yes, from what I can see.

**Pull request closed without merging**